src/main/java/com/fmgs/conferenceapp/controller/PaperController.java

```diff
@@ -0,0 +1,59 @@
+package com.fmgs.conferenceapp.controller;
+
+import com.fmgs.conferenceapp.model.Paper;
+import com.fmgs.conferenceapp.model.Qualifiers;
+import com.fmgs.conferenceapp.model.Review;
+import com.fmgs.conferenceapp.repository.PaperRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Map;
+
+@RestController
+public class PaperController {
+
+    private PaperRepository repository;
+    private Review reviewCtrl;
+
+    @Autowired
+    public PaperController(PaperRepository repository) {
+        this.repository = repository;
+    }
+
+    @GetMapping("/papers")
+    @CrossOrigin(origins = "*")
+    public List<Paper> getAllPapers() {
+        return repository.findAll();
+    }
+
+    @PostMapping("papers/post")
+    @CrossOrigin(origins = "*")
+    public void postPaper(@RequestBody Paper paper) {
+        repository.save(paper);
+    }
+
+    @PostMapping("papers/review")
+    @CrossOrigin(origins = "*")
+    public Paper reviewPaper(@RequestBody Map<String, String> params, String reviewerID, int reviewInt){
```

Comment:
De ce ai ai request body si parametrii la funcție? E ori una, ori cealaltă. Dar nu o sa pui parametrii in amble locuri